make/config.mk

```diff
@@ -48,6 +48,15 @@ USE_CUDA_PATH = NONE
 # whether use CuDNN R3 library
 USE_CUDNN = 0
 
+# CUDA architecture setting: going with all of them.
+# For CUDA < 6.0, comment the *_50 lines for compatibility.
+CUDA_ARCH := -gencode arch=compute_20,code=sm_20 \
```

Comment:
I think this is necessary, and `arch=compute_20,code=sm_20`, `arch=compute_20,code=sm_21` shoud be removed from the default config.mk, It will be depreacted in the future.